github/github_test.go

```diff
@@ -105,6 +105,16 @@ func testHeader(t *testing.T, r *http.Request, header string, want string) {
 	}
 }
 
+func testCanonicalHeader(t *testing.T, r *http.Request, header string, want string) {
+	canonical := http.CanonicalHeaderKey(header)
+	for _, h := range r.Header[canonical] {
+		if h == want {
+			return
+		}
+	}
+	t.Errorf("Multiple header get, wanted %s but not found for header %s", want, header)
+}
```

Comment:
My understanding was to keep the original `Accept` header and add the new one per the [documentation on the endpoint](https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button). So that's why I added this function in even though it is temporary.

So I will defer the choice to keep/rename the function I added or adjust the function as you suggested.